remme/rest_api/openapi.yml

```diff
@@ -165,6 +165,31 @@ paths:
           $ref: '#/responses/404NoCertificate'
         500:
           $ref: '#/responses/500GeneralError'
+  /certificate/store:
+    put:
+      operationId: remme.rest_api.certificate.store
+      summary: Signes and stores provided certificate to REMchain
+      parameters:
+        - $ref: '#/parameters/certificate_payload'
+      responses:
+        200:
+          description: Certificate is registered on the REMchain
+          schema:
```

Comment:
From now on I propose not to use schema descriptions nested into responses or parameters descriptions. It is required by several tools used for generating API docs and I think we should follow those rules. Will line them out later.